lib/handlers/patch.js

```diff
@@ -60,7 +62,19 @@ async function patchHandler (req, res, next) {
     const pathSplit = path.split('/')
     const directory = pathSplit.slice(0, pathSplit.length - 1).join('/')
     if (!fs.existsSync(directory)) {
-      fx.mkdirSync(directory, { recursive: true })
+      const firstDirectoryRecursivelyCreated = await promisify(mkdirp)(directory)
```

Comment:
The mkdirp function https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir.md doesn't seem to return the directory, only an error code. Are you below  in 69 assuming it firstDirectoryRecursivelyCreated will be the directory name?

Reply:
I removed fs extra from node solid server as this seemed to be the only place it was used. I replaced it with https://www.npmjs.com/package/mkdirp which does return the directory.